alfalfa_worker/jobs/openstudio/step_run.py

```diff
@@ -35,9 +35,9 @@ def __init__(self, run_id, realtime, timescale, external_clock, start_datetime,
 
         # EnergyPlus MLEP initializations
         self.ep = mlep.MlepProcess()
-        self.ep.bcvtbDir = '/home/alfalfa/bcvtb/'
-        self.ep.env = {'BCVTB_HOME': '/home/alfalfa/bcvtb'}
-        self.ep.accept_timeout = 120000
+        self.ep.bcvtbDir = '/alfalfa/bcvtb/'
+        self.ep.env = {'BCVTB_HOME': '/alfalfa/bcvtb'}
+        self.ep.accept_timeout = 5000
```

Comment:
We increased this timeout intentionally from one minute to two when working with a complex E+ model for BUILD-SOS.  Is there a reason you are confident that this initialization will only take 5 seconds now?

Reply:
Yeah, so the change here is that instead of doing the accept once it will retry for `self.model_start_timeout` seconds. So while the `ep.accept_timeout` is only 5 seconds it will repeatedly retry for 5 minutes. This means that if the model has a catastrophic error and produces a `.err` file the user only has to wait 5 seconds before seeing the reason the model failed instead of the entire timeout period.

Reply:
Maybe I should add comments to those lines

Reply:
love it!